src/coordinates.cpp

```diff
@@ -1,5 +1,6 @@
 #include "phes_base.h"
 
+// Initializes a GeographicCoordinate with given latitude and longitude
 GeographicCoordinate GeographicCoordinate_init(double latitude, double longitude)
 {
 	GeographicCoordinate geographic_coordinate;
```

Comment:
This should be modernized to proper C++ and write a GeographicCoordinate constructor